samples/notebooks/python/02-running-prompts-from-file.ipynb

```diff
@@ -1,190 +1,191 @@
 {
- "cells": [
```

Comment:
Would be good to undo the whitespace changes, up to you.

Reply:
whitespacing resolved